modules/auxiliary/scanner/portscan/syn.rb

```diff
@@ -62,6 +72,9 @@ def run_batch(hosts)
 
         self.capture.setfilter(getfilter(shost, sport, dhost, dport))
 
+        # Add the delay based on JITTER and DELAY if needs be
+        add_delay_jitter(delay_value,jitter_value)
```

Comment:
Similar to my other comments about moving most of this code to the Scanner mixin, I feel like this could or should go there too.  This particular part may be a bit harder.